tests/framework/extensions/kubeapi/rbac/create.go

```diff
@@ -113,3 +113,78 @@ func CreateGlobalRoleBinding(client *rancher.Client, clusterName string, globalR
 
 	return newGlobalRoleBinding, nil
 }
+
+// CreateClusterRoleTemplateBinding is a helper function that uses the dynamic client to create a cluster cluster role template binding for a specific user
+// in the given downstream cluster.
+func CreateClusterRoleTemplateBinding(client *rancher.Client, clusterName, namespace, crtbName, userName, roleTemplateName, principalID string, annotations, labels map[string]string) (*v3.ClusterRoleTemplateBinding, error) {
+	dynamicClient, err := client.GetDownStreamClusterClient(clusterName)
```

Comment:
Since CRTBs are created only on local cluster, can we remove the clusterName function parameter and use const localcluster within GetDownstreamClusterClient?